sdk/ai/azure-ai-inference/apiview-properties.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
I don't think this file should be committed. It's not needed, you can delete it.